docs/dm/zh_CN/features/column-mapping.md

```diff
@@ -29,36 +36,37 @@ column-mappings:
 ​    arguments: ["2", "test_", "t_"]
 ```
 
-#### 参数解释
+### 参数解释
 
 - [`schema-pattern` / `table-pattern`](./table-selector.md): 对匹配上该规则的上游 MySQL/MariaDB 实例的表按照指定 `expression` 进行列值修改操作
 - `source-column`，`target-column`：对 `source-column` 列的值按照指定 `expression` 进行修改，将修改后的值赋值给 `target-column`
 - `expression`: 对数据进行转换的表达式，目前只支持下面的内置计算表达式
 
-##### `partition id` 
+#### partition id
 
 `partition id` 目的是为了解决分库分表合并同步的自增主键的冲突
 
-###### 使用注意
+##### 使用注意
 - 只支持类型为 bigint 的自增主键
 - 库名的组成必须为 `schema 前缀 + 数字（既 schema ID）`，例如: 支持 `s_1`, 不支持 `s_a`
 - 表名的组成必须为 `table 前缀 + 数字（既 table ID）`
 - 对分库分表的规模支持限制如下
   - 支持最多 15 个 MySQL/MariaDB 实例（instance ID ）
-  - 每个实例支持最多 127 个 schema（schema ID）
-  - 每个实例的每个 schema 255 个 table（table ID）
-  - 自增主键 ID 范围 (>= 0, <= 17592186044415)
+  - 每个实例支持最多 127 个 schema（0 <= schema ID  <= 127）
```

Comment:
Is `schemaID = 0` supported? If we support 127 schemas, better 0 < schema ID <= 127?